packages/cogify/src/cogify/cli/cli.cover.ts

```diff
@@ -75,15 +76,15 @@ export const BasemapsCogifyCoverCommand = command({
 
     const res = await createTileCover(ctx);
 
-    const targetPath = fsa.joinAll(args.target, String(tms.projection.code), im.name, CliId);
+    const targetPath = new URL(`${tms.projection.code}/${im.name}/${CliId}/`, args.target);
```

Comment:
Does this need to be a `UrlFolder` as you defined in the args, or you just need the args.target to be end with '/'

Reply:
`UrlFolder` isn't it's own type, it's a parser which takes the string from the CLI and turns it into a `URL` while ensuring it ends with a slash. So the type of `args.target` here is `URL`.